graphql/src/main/java/com/linecorp/armeria/server/graphql/GraphqlServiceBuilder.java

```diff
@@ -83,7 +92,39 @@ public GraphqlServiceBuilder schemaFile(File... schemaFiles) {
      * If not set, the {@code schema.graphql} or {@code schema.graphqls} will be imported from the resource.
      */
     public GraphqlServiceBuilder schemaFile(Iterable<? extends File> schemaFiles) {
-        this.schemaFiles.addAll(requireNonNull(schemaFiles, "schemaFiles"));
+        requireNonNull(schemaFiles, "schemaFiles");
+        return schemaUrls(Streams.stream(schemaFiles)
+                                 .map(file -> {
+                                     try {
+                                         return file.toURI().toURL();
+                                     } catch (MalformedURLException e) {
+                                         throw new UncheckedIOException(e);
+                                     }
+                                 }).collect(toImmutableList()));
+    }
+
+    /**
+     * Adds the schema {@link URL}s.
+     * If not set, the {@code schema.graphql} or {@code schema.graphqls} will be imported from the resource.
+     */
+    public GraphqlServiceBuilder schemaUrls(URL... schemaUrls) {
```

Comment:
It would be convenient to take path strings as arguments.
You can refer to:
https://github.com/line/armeria/blob/5b384fbe27e7e6f9225d6db91cbb684d09dfbb5e/dropwizard2/src/main/java/com/linecorp/armeria/dropwizard/ArmeriaConfigurationUtil.java#L409-L423

Reply:
We can introduce an internal utility use it in both places.

Reply:
Oh... I didn't know I was using this.

That's a good way. 👍